hoomd/hpmc/UpdaterMuVT.h

```diff
@@ -312,7 +312,8 @@ UpdaterMuVT<Shape>::UpdaterMuVT(std::shared_ptr<SystemDefinition> sysdef,
 
     if (m_sysdef->getNDimensions() == 2)
```

Comment:
Remove the whole check. The constructing message has already been printed.

Reply:
👍